src/pods/canvas/canvas.vm.ts

```diff
@@ -47,3 +47,14 @@ export interface XRelationCoords {
   xOrigin: number;
   xDestination: number;
 }
+
+export interface YRelationCoords {
```

Comment:
I think we can do the same with this interface as with `SeekResult`, but we will need to export this interface

Reply:
It's fixed